src/sections/hero.html

```diff
@@ -9,13 +9,13 @@
       </div>
     <div class="lg:h-[64.208rem] lg:mx-auto relative -mt-52 lg:mt-20">
     <div class="w-full relative h-screen md:min-h-screen lg:mt-0">
-        <img src="./src/images/green.svg" alt="green shape" class="z-[5] lg:z-[4] absolute bottom-[17%] md:bottom-[29%] left-[3%] lg:top-[14.341%] lg:left-[20.54%] w-[56%] lg:w-[34.61%]"/>
+        <img src="./src/images/green.svg" alt="green shape" class="z-[5] lg:z-[4] absolute bottom-[17%] md:bottom-[29%] left-[3%] lg:top-[14.341%] lg:left-[20.54%] w-[56%] lg:w-[34.65%]"/>
         <img src="./src/images/red.svg" alt="red shape" class="z-[4] lg:z-[3] absolute bottom-[4.7%] md:bottom-[8.2%] right-[5%] lg:left-[13.04%] lg:top-[49.58%] w-[73.5%] lg:w-[47.66%]"/>
         <img src="./src/images/blue.svg" alt="blue shape" class="z-[3] lg:z-[5] absolute bottom-[11%] md:bottom-[19%] lg:top-[6.7%] w-[70%] lg:w-[46.4993395%] right-[5%] lg:left-[41.22%]"/>
         <img src="./src/images/yellow.svg" alt="yellow shape" class="z-[2] absolute left-0 bottom-[7.5%] md:bottom-[12%] lg:top-0 lg:left-0 w-[76%] lg:w-[47.95%]"/>
         <img src="./src/images/right-green.svg" alt="Green shape" class="z-[-10] absolute bottom-0 lg:right-0 lg:top-[24.42%] right-0 w-[42%] lg:w-[26.82%] "/>
         <img src="./src/images/pink.svg" alt="Pink shape" class="absolute bottom-[26%] md:bottom-[43%] right-[9%] w-[30%] lg:w-[15.59%] lg:left-[39.56%] lg:top-[2.27%]"/>
-        <div id="paragraphDiv" class="absolute z-30 inset-0 lg:flex justify-center lg:items-center md:hidden top-[60%] lg:top-0">
+        <div class="absolute z-30 inset-0 lg:flex justify-center lg:items-center md:hidden top-[58%] lg:top-[30%]">
```

Comment:
The text is too far down on the large screen breakpoint.

Reply:
Resolved